feat: support session cookies

DanielRivers opened the pull request:
# Explain your changes

Add support for `ksp` to enable session cookies

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

DanielRivers committed:
feat: support session cookies

DanielRivers committed:
docs: update jsdocs

dtoxvanilla1991 commented:
@copilot can you resolve branch conflicts with master branch and make sure the build passes? Do not open a new PR, it must be done in this same PR.

dtoxvanilla1991 commented:
I'll take over this PR.

Copilot committed:
chore: merge main into feat/session-cookies, resolving conflicts

Co-authored-by: dtoxvanilla1991 <73205087+dtoxvanilla1991@users.noreply.github.com>

Copilot committed:
fix: update tests to match merged commitTokenToSession signature

Agent-Logs-Url: https://github.com/kinde-oss/kinde-typescript-sdk/sessions/1ea97928-61ad-4eec-bd1c-71912b9e5c32

Co-authored-by: dtoxvanilla1991 <73205087+dtoxvanilla1991@users.noreply.github.com>

dtoxvanilla1991 commented:
PR has no diff. Closing. 📪
